abstutil/src/logger.rs

```diff
@@ -1,74 +1,16 @@
-use std::sync::RwLock;
-
-use instant::Instant;
-
-use crate::elapsed_seconds;
-
-pub struct Logger {
-    last_fast_paths_note: RwLock<Option<Instant>>,
-}
-
-impl Logger {
-    /// On native: intercept messages using the `log` crate and print them to STDOUT. Contains
-    /// special handling to filter/throttle spammy messages from `fast_paths` and `hyper`.
-    ///
-    /// On web: Just use console_log.
-    pub fn setup() {
-        #[cfg(target_arch = "wasm32")]
-        {
-            console_log::init_with_level(log::Level::Info).unwrap();
-        }
-
-        #[cfg(not(target_arch = "wasm32"))]
-        {
-            log::set_boxed_logger(Box::new(Logger {
-                last_fast_paths_note: RwLock::new(None),
-            }))
-            .unwrap();
-            log::set_max_level(log::LevelFilter::Info);
-        }
+/// On native: intercept messages using the `log` crate and print them to STDOUT. Contains
```

Comment:
Update docstring to something like: Initialize `env_logger` on native and `console_log` on web.

Reply:
Done - plus added some examples. It's mostly duplicated w/ what I added to the dev docs, but I don't know where people are more likely to look. Happy to remove one if you'd prefer to keep it DRY.